x/epochs/keeper/abci_test.go

```diff
@@ -10,146 +10,98 @@ import (
 	simapp "github.com/osmosis-labs/osmosis/v7/app"
 	"github.com/osmosis-labs/osmosis/v7/x/epochs/types"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
-func TestEpochInfoChangesBeginBlockerAndInitGenesis(t *testing.T) {
-	var app *simapp.OsmosisApp
-	var ctx sdk.Context
-	var epochInfo types.EpochInfo
+	"golang.org/x/exp/maps"
 
-	now := time.Now()
+	"github.com/osmosis-labs/osmosis/v7/osmoutils"
+)
 
-	tests := []struct {
-		expCurrentEpochStartTime   time.Time
-		expCurrentEpochStartHeight int64
-		expCurrentEpoch            int64
-		expInitialEpochStartTime   time.Time
-		fn                         func()
+// This test is responsible for testing how epochs increment based off
+// of their initial conditions, and subsequent block height / times.
+//
+// TODO: Make a new test for init genesis logic
+func (suite KeeperTestSuite) TestEpochInfoBeginBlockChanges() {
+	block1Time := time.Unix(1656907200, 0).UTC()
+	// We run all tests with the same identifier and same duration
+	defaultIdentifier := "hourly"
+	defaultDuration := time.Hour
+	eps := time.Nanosecond
+
+	tests := map[string]struct {
+		// if identifier, duration is not set, we make it defaultIdentifier and defaultDuration.
```

Comment:
Were `identifier` and `duration` removed? I don't see anything other than default

Reply:
ah yeah, didn't write a test with a different one. You made a good point, will do it

Reply:
Added!